packages/db/src/models/collection.ts

```diff
@@ -12,6 +13,32 @@ class CollectionPgModel extends BasePgModel<PostgresCollection, PostgresCollecti
     });
   }
 
+  /**
+   * Fetches multiple items from Postgres
+   *
+   * @param {Knex | Knex.Transaction} knexOrTransaction - DB client or transaction
+   * @param {Partial<RecordType>} params - An object or any portion of an object of type RecordType
+   * @param {updatedAtRange} updatedAtParams - An object with Date search bounds for updatedAt
+   * @returns {Promise<RecordType[]>} List of returned records
+   */
+  async searchWithUpdatedAtRange(
+    knexOrTransaction: Knex | Knex.Transaction,
```

Comment:
I debated making this a base model method, but the base model doesn't assume timestamps.   Open to suggestions on drying this up - we could make it a /lib helper that takes any type with the whereBetween params, but I landed on YAGNI at least until the granule work.

Reply:
> The base model doesn't assume timestamps

You mean we can't assume the presence of them from the types? But don't all of our data types have `updated_at` columns based on our schema definitions. I'm pretty sure that they do

Reply:
You're right that everything that's *currently*  extending BasePgModel does, with GranulesExecutionsPgModel (which doesn't inherit) being the exception in ./models.

So we could make it a method on the base model, but then that means we should update the base class from 

`extends { cumulus_id: number }`

to 

`extends { cumulus_id: number, updated_at: Date, created_at: Date }` 

but that's not *currently* the requirement.

Reply:
I'm kinda inclined to add `updated_at` and `created_at` to that type constraint and move to this base model. I'm not seeing a downside and it is convenient. Thoughts?

Reply:
I would have said it's an issue for GranulesExecutions, but it seems that doesn't extend the base model, so..... so long as we're comfortable with all record classes having those values and extending the base model.     See what you think of 4b6b254ef